src/api/app/controllers/webui/users/tasks_controller.rb

```diff
@@ -0,0 +1,14 @@
+module Webui
+  module Users
+    class TasksController < WebuiController
+      before_action :require_login
+      before_action :check_display_user
+
+      def index
+        redirect_back(fallback_location: root_path, notice: 'Sorry you\'re not allowed to view tasks of other users') unless @is_displayed_user
```

Comment:
I'm wondering if we even need to do that?
Why not just use ``User.current``? And the route would also be easier ``/user/tasks``, no need to add the login as a parameter.